R/mcmc.R

```diff
@@ -766,3 +766,24 @@ setMethod("mcmc",
     }
 )
 # nolint end
+
+## -----------------------------------------------------------------------------------
+## obtain the posterior samples for ordinal models
+## ----------------------------------------------------------------------------
```

Comment:
please avoid these delimiters - you can use `ordinal ----` instead e.g. (we might have some old delimiters in this file but please ignore those)